lib/badfruit/Movies/movie.rb

```diff
@@ -26,6 +26,11 @@ def full_cast
       return @badfruit.parse_actors_array(JSON.parse(@badfruit.get_movie_info(@id, "cast")))
     end
 
+    # Returns director, since RottenTomatoes API is broken (abridged_director doesn't show up in search_by_name results for movies.)
+    def director
+      return Director.new(JSON.parse(@badfruit.get_movie_info(@id, "details"))["abridged_directors"][0])
```

Comment:
Okay I've changed it to return an array of directors instead, how does that look?